ember/app/router.js

```diff
@@ -43,5 +45,9 @@ Router.map(function () {
       this.route("add");
       this.route("edit", { path: "/edit/:category" });
     });
+    this.route("membership-roles", { resetNamespace }, function () {
```

Comment:
Since you added a space everywhere else:
```suggestion

    this.route("membership-roles", { resetNamespace }, function () {
```

Reply:
The empty lines currently separate the routes into:
* Mounted engines
* Caluma
* Project Routes